src/bin/latency.rs

```diff
@@ -257,6 +317,10 @@ fn do_client(
                 .expect("Failed to sleep");
         }
         now = SystemTime::now();
+        match oneway {
+            true => perf_pkt.set_op(PerfOp::Tx as u8),
+            false => perf_pkt.set_op(PerfOp::Ping as u8),
+        };
```

Comment:
We don't need to insert `tv_set`, `tv_nsec` after this on Tx/Ping packet. that field is only for Sync packet